src/Payments.sol

```diff
@@ -1293,6 +1418,26 @@ contract Payments is
             ? 0
             : approval.lockupAllowance - oneTimePayment;
     }
+
+    /// @notice Allows the contract owner to withdraw accumulated payment fees.
+    /// @param token The ERC20 token address of the fees to withdraw.
+    /// @param to The address to send the withdrawn fees to.
+    /// @param amount The amount of fees to withdraw.
+    function withdrawFees(
+        address token,
+        address to,
+        uint256 amount
+    ) external onlyOwner nonReentrant validateNonZeroAddress(token, "token") validateNonZeroAddress(to, "to") {
+        require(amount > 0, "amount must be greater than zero");
```

Comment:
nit: I generally prefer to allow "0" in cases like this (makes it easier to write no-op code).

Reply:
Done.